composer.json

```diff
@@ -55,7 +56,8 @@
             "common\\http\\": "common/http",
             "entities\\": "entities",
             "services\\": "services",
-            "repositories\\": "repositories"
+            "repositories\\": "repositories",
+            "OpenEMR\\Encounter\\Services\\": "interface/patient_file/encounter/Services"
```

Comment:
@MatthewVita , just want to get your thoughts on having a agreed upon paths/namespace strategy (if needed) since this seems to be a different strategy from what you are doing.